pkg/apis/core/types.go

```diff
@@ -3813,6 +3829,15 @@ type PodSecurityContext struct {
 	// Note that this field cannot be set when spec.os.name is windows.
 	// +optional
 	AppArmorProfile *AppArmorProfile
+	// seLinuxChangePolicy defines how the container's SELinux label is applied to all volumes used by the Pod.
+	// It has no effect on nodes that do not support SELinux or to volumes does not support SELinux.
+	// Valid values are "MountOption" and "Recursive". If not specified, "MountOption" is used.
+	// It affects only in-tree iSCSI and FibreChannel volumes, and CSI volumes that announce "SELinuxMount: true" in their CSIDriver instance.
+	// It affects only Pods that have SELinux label set, either in PodSecurityContext or in SecurityContext of all containers.
+	// Note that this field cannot be set when spec.os.name is windows.
+	// +featureGate=SELinuxChangePolicy
```

Comment:
Can you also document that `MountOption` is only allowed if the SELinuxMount feature gate is enabled?

Reply:
added